src/elements-experimental/timetable-row/timetable-row.ts

```diff
@@ -170,6 +170,7 @@ const findAndReplaceNotice = (notices: Notice[]): Notice | undefined => {
     if (reservationNotice.includes(notice.name)) {
       return { ...notice, name: 'RR' } as Notice;
     }
+    return undefined;
```

Comment:
why is this not implicitly given?

Reply:
code analysis returns 'TS7030: Not all code paths return a value.'

Reply:
ok. looks odd, but it's ok ;)